### ISSUE TITLE
Papulovesicular eruption

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6782]
**Preferred term label:**
Papulovesicular eruption

**Synonyms**
Papulo-vesicular eruption

**Definition (free text, please give PubMed ID)**
An acute onset rash characterized by by multiple vesicles, which are circumscribed, fluid-containing, epidermal elevations with a diameter less than 10mm at the widest point, and by multiple papules, which are circumscribed, solid epidermal elevations  with no visible fluid with a diameter less than 10mm at the widest point.
PMID:32579185 

**Parent term (use hpo.jax.org/app)**
see vesicular eruption
**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
